pkg/controller/worker/helper.go

```diff
@@ -59,3 +63,30 @@ func (w *workerDelegate) updateWorkerProviderStatus(ctx context.Context, workerS
 		return nil
 	})
 }
+
+func makeCopyOfMap(mp map[string]string) map[string]string {
+	var mpDeepCopy = make(map[string]string)
+	for k, v := range mp {
+		mpDeepCopy[k] = v
+	}
+	return mpDeepCopy
+}
+
+// func addAwsCsiDriverTopologyLabel(machineDeploymentIndex *int, zone string, poolLabels map[string]string, machineDeployments worker.MachineDeployments) {
+// 	if _, exists := poolLabels[AwsCsiDriverTopologyKey]; !exists {
+// 		var poolLabelsDeepCopy = makeCopyOfMap(poolLabels)
+// 		poolLabelsDeepCopy[AwsCsiDriverTopologyKey] = zone
+// 		machineDeployments[*machineDeploymentIndex].Labels = poolLabelsDeepCopy
+// 		*machineDeploymentIndex++
+// 	}
```

Comment:
```suggestion
```
Always remove unwanted code. You can always look up past changes on your git commit history.

Reply:
oh again :/ , sorry , I will keep it mind now